src/MenuScreen.cpp

```
@@ -150,10 +150,13 @@
 
 void MenuScreen::poll(MenuRenderer* renderer, uint16_t pollInterval) {
```

Cppcheck (2.10-2)

[style] The function 'poll' is never used.

(unusedFunction)